content/en/docs/reference/kubectl/extending_kubectl/discovering_plugins.md

```diff
@@ -0,0 +1,56 @@
+---
+title: "Discovering Plugins"
+linkTitle: "Discovering Plugins"
+weight: 2
+type: docs
+description: >
+   Discovering Plugins that suits your requirement
+---
+
+
+{{< alert color="success" title="TL;DR" >}}
+
+- [krew.sigs.k8s.io](https://krew.sigs.k8s.io/) is a kubernetes sub-project to discover and manage plugins
+
+{{< /alert >}}
+
+# Krew
```

Comment:
Although originally this was a pure migration, this will be better if we drop the initial text.
```suggestion
```